docs/source/getting_started.rst

```diff
@@ -114,11 +109,11 @@ Don't change the default file name, which should be *hello_world.json*.
 Just select a folder from your system (it can be the same where you saved the main program file)
 and click **Save**.
 
-Now you should see the new tool specification in the *Project* widget, *Tool specifications* list.
+Now you should see the new tool specification in the *Project* widget, *Tools* list.
```

Comment:
I think this should be: 

    Now you should see the new tool in the *Project* widget, *Tools* list.

Can  you make that change as part of this PR @ridalalven ? After that I think we are ready to merge. Thanks very much for grooming this!